port/common/omrfile.c

```diff
@@ -607,41 +607,33 @@ EsTranslateOpenFlags(int32_t flags)
 void
 omrfile_vprintf(struct OMRPortLibrary *portLibrary, intptr_t fd, const char *format, va_list args)
 {
-	char outputBuffer[256];
-	char *allocatedBuffer;
-	uint32_t numberWritten;
-	va_list copyOfArgs;
-
-	/* Attempt to write output to stack buffer */
-	COPY_VA_LIST(copyOfArgs, args);
```

Comment:
I don't believe it is required. I can't find implementation for str_vprintf(), but I think it handles va_list.